server/src/main/java/org/opensearch/indices/NodeIndicesStats.java

```diff
@@ -281,7 +405,21 @@ public List<IndexShardStats> getShardStats(Index index) {
      *
      * @opensearch.internal
      */
-    static final class Fields {
-        static final String INDICES = "indices";
+    @PublicApi(since = "2.0.0")
```

Comment:
The version value is incorrect.

Reply:
Updated in next revision. [Ref](https://github.com/opensearch-project/OpenSearch/pull/14454/files#diff-b79bb7aee566fa9c63b9f972aaa2b26206e0717a7e7d831307c0794161ec64bfR413)